metadata.yaml

```diff
@@ -27,9 +27,8 @@ containers:
 resources:
```

Comment:
This PR doesn't display this part of the file, so I'll just comment here, but it looks like `display-name` isn't supported in Metadata V2 charms / by charmhub.io:

https://discourse.charmhub.io/t/3674/24

Juju just ignores extra fields, so it won't break anything, but I don't think it will do anything anymore.

Reply:
Thanks for pointing this out. Removed.